test/TestUtilities/Test.Utility/TestSources.cs

```diff
@@ -16,5 +19,11 @@ public static class TestSources
         public const string ProGet = nameof(ProGet);
         public const string TeamCity = nameof(TeamCity);
         public const string VSTS = nameof(VSTS);
+
+        public static string GetConfigFileRoot()
+        {
+            var fullPath = Environment.GetEnvironmentVariable("NuGet_Core_FuncTests_Config");
```

Comment:
NuGet_FuncTests_Config maybe?
There are 3 configs that are used...NuGet.Config, NUGet.Core.FuncTests.Config, NuGet.Protocol.FunctTest.config